c++/src/capnp/rpc.c++

```diff
@@ -784,7 +799,8 @@ private:
                    kj::Array<PipelineOp>&& ops)
         : RpcClient(connectionState), questionRef(kj::mv(questionRef)), ops(kj::mv(ops)) {}
 
-   kj::Maybe<ExportId> writeDescriptor(rpc::CapDescriptor::Builder descriptor) override {
+   kj::Maybe<ExportId> writeDescriptor(rpc::CapDescriptor::Builder descriptor,
+                                       kj::Vector<int>& fds) override {
```

Comment:
I assume it's intentional that these go unused here?

Reply:
Yes, a PipelineClient represents a capability that is expected to be returned by an RPC call which is at present still in-flight. So we clearly can't yet be aware of any file descriptors attached to it.